DuckDuckGo/AppDelegate.swift

```diff
@@ -519,6 +519,7 @@ import os.log
             ContentBlocking.shared.contentBlockingManager.scheduleCompilation()
             AppConfigurationFetch.shouldScheduleRulesCompilationOnAppLaunch = false
         }
+        AppDependencyProvider.shared.configurationManager.loadPrivacyConfigFromDiskIfNeeded()
```

Comment:
Should this happen before `AppDependencyProvider.shared.configurationManager` is used in line 295?

Reply:
It doesn't matter. We don't use the built in timer of the configuration manager because of background app refresh. And even if we did the only purpose of this method is to diff the stored etag vs the etag in the privacy config manager and load fresh from disk if needed.